lib/Redmine/Api/AbstractApi.php

```diff
@@ -204,7 +204,10 @@ protected function attachCustomFieldXML(\SimpleXMLElement $xml, array $fields)
                     $_values->addChild('value', $val);
                 }
             } else {
-                $_field->addChild('value', $field['value']);
+                // as in Issue::buildXML method
```

Comment:
could you remove those comments please?

Anyway you could unit test this?

Reply:
Thank you @kbsali for your reply.

Of course I will remove comments...it was just to refer an existing and similar case.
I will try to unit test it next week.

How long it will take after unit test to set it available into next 1.5.xx  version?

Reply:
not long, i promise :)